data_dir/mv_synchronous_updates_on_standard.yaml

```diff
@@ -0,0 +1,22 @@
+# LWT test: create and update data using LWT.
+### DML ###
+
+# Keyspace Name
+keyspace: keyspace1
+
+# Table name
+table: standard1
+
+#
```

Comment:
I don't understand where is the MV creation for standard1....

Reply:
Now, when it's in the same PR it confused me, and I still don't understand why it happens in the code and not here.
So, if it has a good reason, explain it in a comment here and mention where it's located in the code...

Reply:
I want to create MV as part of the table data, I need to be able to apply filter in create MV statement.
Column types in the "standard1" table are "blob". So I decided to use tokens. I can not do it in the user profile. this is the reason why I created the main MV in the code and not in the user profile.

Reply:
1. Do we really need filter? (isn't it covered also somewhere else)
2. If so, as I asked above, please add comment here where the MV creation for this table.